### ISSUE TITLE
Can cfonts be webpacked?

### ISSUE THREAD

**Issue by rushairer** [dominikwilkowski/cfonts#22]
When I build my project with webpack,then run the .js file, I got error:

```
Error: ENOENT: no such file or directory, open '/package.json'
    at Object.openSync (fs.js:443:3)
    at Object.readFileSync (fs.js:343:35)
```

Can webpack cfonts into one file? Thanks.

**Comment by dominikwilkowski** [comment]
Hey @rushairer 

Can you tell me a little bit about what you're trying to do? CFonts is a console utility and I'm not sure how it would work in the browser.

**Comment by rushairer** [comment]
I changed some codes at https://github.com/rushairer/cfonts

I use cfonts in a console application, but I 'd like pack the codes into one file.

**Comment by dominikwilkowski** [comment]
@rushairer with those changes (fixing some path resolutions) it now works in webpack?

**Comment by rushairer** [comment]
@dominikwilkowski 
Yes, it works well. Thanks for your amazing cfont, it's very cool~!

**Comment by dominikwilkowski** [comment]
Great. Let me make that fix here and push a new version out. If it helps and works why not :)

**Comment by akupiec** [comment]
Unfortunately at version 2.8.1 webpack is still unable to correctly bundle package.json, it seems like was reintroduced again by commit: 
https://github.com/dominikwilkowski/cfonts/commit/9238bb79b1955df2269edbcf0a21a1eb7aa0aa1b

Currently Im using workaround with string-replace-loader at webpack config, but it would be great it this could be fixed :D

`
module.exports = {
  entry: './src/index.ts',
  target: 'node',
  mode: 'production',
  devtool: false,
  module: {
    rules: [{ test: /\.tsx?$/, loader: 'ts-loader', exclude: /node_modules/ },
      {
        test: /node_modules\/cfonts.*constants\.js$/,
        loader: 'string-replace-loader',
        options: {
          search: 'path.normalize(`${__dirname}/../package.json`)',
          replace: `'../package.json'`,
        }
      }],
  },
  resolve: {
    extensions: ['.tsx', '.ts', '.js'],
  },
`

**Comment by dominikwilkowski** [comment]
Fixed in v2.8.2

Was an issue with the typescript types and needed `--resolveJsonModule`

Thanks for the report @akupiec
